packages/palette/src/elements/Spacer/Spacer.story.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import { Flex } from "../Flex"
+import { Text } from "../Text"
+import { Spacer, SpacerProps } from "./Spacer"
+
+export default {
+  title: "Components/Spacer",
+  component: Spacer,
+}
+
+const SpacerRow = (props: { x: SpacerProps["x"] }) => {
+  const size = props.x
+
+  return (
+    <Flex mb={1}>
+      <Flex flexDirection="row" alignItems="center">
+        <Flex width={20} height={20} backgroundColor="black100" />
+        <Spacer {...props} />
+        <Flex width={20} height={20} backgroundColor="black100" />
+      </Flex>
+      <Text ml={0.5}>{size}</Text>
+    </Flex>
+  )
+}
+
+export const Horizontal = () => (
+  <Flex flexDirection="column">
+    <Text variant="lg">Defined (units)</Text>
+    <SpacerRow x={0.5} />
+    <SpacerRow x={1} />
+    <SpacerRow x={2} />
+    <SpacerRow x={4} />
+    <SpacerRow x={6} />
+    <SpacerRow x={12} />
+
+    <Text variant="lg">Custom (pixels/em)</Text>
+    <SpacerRow x="15px" />
+    <SpacerRow x="50px" />
+    <SpacerRow x="6em" />
+  </Flex>
+)
```

Comment:
Can you switch these stories to use the `States` component from `storybook-states`. See other stories for examples.